pkgs/by-name/tu/tuist/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tuist";
+  version = "4.38.2";
+
+  src = fetchurl {
+    url = "https://github.com/tuist/tuist/releases/download/${finalAttrs.version}/tuist.zip";
```

Comment:
As the source is available, please download that and compile from source instead

Reply:
It didn't work because we have an old version of Swift

Reply:
Could I ask what version of swift it needs?

It shouldn't be a problem to update the version of swift in `nixpkgs` unstable, then build from source using that. It needs doing anyhow if it is out of date, so it is preferable to importing a binary blob

Reply:
Tuist needs [Swift 5.10](https://github.com/tuist/tuist/blob/7250ff3bac2969e20385b78606e3e9128483298f/Package.swift#L1), [nixpkgs is at 5.8](https://github.com/NixOS/nixpkgs/blob/88195a94f390381c6afcdaa933c2f6ff93959cb4/pkgs/development/compilers/swift/sources.nix#L8)

Reply:
Fair enough, not much else you can do!

I'll see if I can lend a hand with a PR for Swift 6, it seems crazy to be importing binary blobs due to that derivation being stuck